projects/storefrontlib/src/lib/ui/pages/store-finder-page/store-finder-page.module.ts

```diff
@@ -1,66 +0,0 @@
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-import { RouterModule, Routes } from '@angular/router';
-
-import { CmsPageGuard } from '../../../cms/guards/cms-page.guard';
-import { StoreFinderPageComponent } from './store-finder-page.component';
-import { StoreFinderPageLayoutModule } from '../../layout/store-finder-page-layout/store-finder-page-layout.module';
-// tslint:disable-next-line:max-line-length
-import { StoreFinderSearchResultComponent } from '../../../store-finder/components/store-finder-search-result/store-finder-search-result.component';
-// tslint:disable-next-line:max-line-length
-import { StoreFinderStoresCountComponent } from '../../../store-finder/components/store-finder-stores-count/store-finder-stores-count.component';
-import { StoreFinderGridComponent } from '../../../store-finder/components/store-finder-grid/store-finder-grid.component';
-// tslint:disable-next-line:max-line-length
-import { StoreFinderStoreDescriptionComponent } from '../../../store-finder/components/store-finder-store-description/store-finder-store-description.component';
-
-const routes: Routes = [
```

Comment:
Now we can also remove `storeFinder` from `default-storefront-routes-translations.ts`

Reply:
👍